internal/aio/aio_dst.go

```diff
@@ -95,27 +95,25 @@ func (a *aioDST) Flush(t int64) {
 		if subsystem, ok := a.subsystems[sqes.Key]; ok {
 			// Randomly decide whether to process SQE or return an error
 			if a.r.Float64() < a.failureProbability {
-				// Create a new CQE
-				errorCqe := createErrorCQE("aio dst error occurred while processing SQE", "aio dst failure", simpleCallback)
-				errorCqe.Completion = &t_aio.Completion{}
-
-				slog.Debug("aio dst: failure", "err", errorCqe)
-				a.cqes = append(a.cqes, errorCqe)
-			} else {
 				// Do the I/O
 				processedCQEs := subsystem.NewWorker(0).Process(sqes.Value)
 				// Randomly decide whether to return an error after processing SQE
-				if a.r.Float64() < a.failureProbability {
-					for _, cqe := range processedCQEs {
-						errorCqe := createErrorCQE("aio dst error occurred after processing SQE", "aio dst failure", simpleCallback)
-						errorCqe.Completion = &t_aio.Completion{}
-
-						slog.Debug("aio dst: failure", "err", errorCqe, "cqe", cqe)
-						a.cqes = append(a.cqes, cqe, errorCqe)
+				for i := range processedCQEs {
+					processedCQEs[i].Error = errors.New("aio dst: failure, after processing")
+				}
+				a.cqes = append(a.cqes, processedCQEs...)
+			} else {
+				// error without processing
+				res := make([]*bus.CQE[t_aio.Submission, t_aio.Completion], len(sqes.Value))
+				for i := range sqes.Value {
+					res[i] = &bus.CQE[t_aio.Submission, t_aio.Completion]{
+						Metadata:   sqes.Value[i].Metadata,
+						Completion: nil,
+						Callback:   sqes.Value[i].Callback,
+						Error:      errors.New("aio dst: failure, before processing"),
```

Comment:
can we make this and the other error variables above the file: 

ErrFailureInjectionBeforeAIOStoreProcessing = errors.New("")
ErrFailureInjectionAfterAIOStoreProcessing = errors.New("")